backend/image/migrations/0001_initial.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.24 on 2024-03-06 18:27
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Image',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('file', models.ImageField(upload_to='images/')),
+                ('title', models.CharField(max_length=255)),
+                ('description', models.TextField()),
+                ('alt', models.CharField(max_length=255)),
+                ('href', models.URLField()),
+                ('rel', models.CharField(max_length=255)),
+                ('target', models.CharField(max_length=255)),
```

Comment:
I think it would be safer/easier to override the `target`  field in the form with a choicefield, as there are only a few valid options.

Reply:
Good point! I actually thought about that before ... and then I forgot about it again 😄